docs/docs/1.2.3/policies/traffic-metrics.md

```diff
@@ -136,71 +136,75 @@ metrics:
     path: /non-standard-path
 ```
 
-As a result, this particular dataplane will expose an HTTP endpoint with `Prometheus` metrics on port `1234` and URI path `/non-standard-path`.
+This proxy exposes an HTTP endpoint with Prometheus metrics on port `1234` and URI path `/non-standard-path`.
 :::
 ::::
 
 ### Configure Prometheus
 
-Although dataplane metrics are now exposed, `Prometheus` doesn't know anything about it just yet.
+Although proxy metrics are now exposed, you still need to let Prometheus discover them.
 
-To help `Prometheus` to automatically discover dataplanes, `Kuma` provides a tool - `kuma-prometheus-sd`.
-`kuma-prometheus-sd` is meant to run alongside `Prometheus` instance.
-It knows location of `Kuma` Control Plane is and can fetch an up-to-date list of dataplanes from it.
-It then transforms that information into a format that `Prometheus` can understand, and saves it into a file on disk.
-`Prometheus` watches for changes to that file and updates its scraping configuration accordingly.
+In Prometheus version 2.29 and later, you can add Kuma metrics to your `prometheus.yml`:
+
+```sh
+kuma_sd_configs:
+ - server: http://kuma-control-plane.kuma-system.svc:5676
+```
```

Comment:
Really feels you need to be a little bit more explicit about this.

My intuition would be to link to the [prometheus doc](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#kuma_sd_config) (especially because we're talking about prometheus). Though I know you don't link that because of maintenance of links.

If you don't link you should at least give a full scrape config:

```yaml
scrape_configs:
    - job_name: 'kuma-dataplanes'
      scrape_interval: "5s"
      kuma_sd_configs:
      - server: "http://kuma-control-plane.kuma-system.svc:5676"
```

This would help understand where this configuration should fit.

Reply:
How about both? (This is a place where the external link is worth the potential 404 hit -- and a data point toward implementing link checker ...)